src/main/java/org/swisspush/redisques/handler/RedisquesHttpRequestHandler.java

```diff
@@ -107,13 +55,22 @@ public class RedisquesHttpRequestHandler implements Handler<HttpServerRequest> {
     private static final String EMPTY_QUEUES_PARAM = "emptyQueues";
     private static final String DELETED = "deleted";
 
+    /**
+     * <p>For why we should NOT use such date formats, see SDCISA-15311. We really
```

Comment:
Does not really make sense IMO, to link to an internal issue in an opensource library...

Reply:
Unluckily I see no other way to refer to this content. I cannot make it public. The only option I have is to refer to internal stuff. So at least SOME devs can follow what I mean. Dropping it will make it harder to understand for internal coworkers too :(